spec/features/hub/show_client_spec.rb

```diff
@@ -72,9 +72,11 @@
   end
 
   skip "user without admin access, but is coalition lead for client organization" do
-    let(:user) { create :user, supported_organizations: [client.vita_partner, other_vita_partner] }
+    # this test may not be entirely correct
```

Comment:
It seems correct to me! Why the comment? :D

Reply:
well i don't entirely know why it was skipped in the first place and it's not like i made it pass or anything so i wanted the next person to know that this test may never have been correct to begin with?

Reply:
Makes sense!

@alaroia-cfa and I took a look and clarified the spec in this follow-up commit, fyi: c8eccadb